docs/rfcs/004-issueish-list.md

```diff
@@ -0,0 +1,106 @@
+# Issueish List
+
+## Status
+
+Proposed
+
+## Summary
+
+Display a list of all open pull requests in the current repository in the GitHub tab.
+
+## Motivation
+
+To provide a navigational element that makes sense even if you aren't on an active feature branch.
+
+To give users a way to see an overview of what's going on in the repository.
+
+As an initial building block toward a pull request review workflow.
+
+## Explanation
+
+### Accordion Lists
+
+Within the GitHub panel, render a vertical stack of two collapsible lists of _issueish_ (pull request or issue) items:
+
+_First list: current pull request_. If the active branch is associated with one or more open pull requests on the GitHub repository, render an item for each.
+
+_Second list: all open pull requests_. List all open pull requests on the GitHub repository.
+
+Each list has a "collapse arrow" in its header. Clicking the collapse arrow toggles the visibility of that list's items, accordion-style.
+
+If either list exceeds 20 items, truncate the list and render a "More" link after its final item. Clicking "more" opens the corresponding search on GitHub.
+
+![Lists](https://user-images.githubusercontent.com/378023/40964722-ceb9d95a-68e6-11e8-90b3-1c155cdc2c00.png)
+
+Each list item renders a tile containing a compact set of information about that pull request:
+
+* Mini author avatar
+* Title, truncated if necessary
+* Terse relative timestamp (1d, 2h, 30m)
+* Pull request state: open, closed, merged
+* Status check summary
+
+![List item](https://user-images.githubusercontent.com/378023/40964791-f4b28fbc-68e6-11e8-907b-c7d436d0d315.png)
+
+Clicking on a list item opens an issueish pane item for the chosen issueish in the same pane container as the parent component (by default, the right dock). If the issuish pane item is already open, it is activated instead.
+
+### Issueish Pane Item: Pull Request
+
+For a pull request, the issueish pane shows:
+
+* Author avatar
+* Title
+* Link to .com. -> [atom/github#1503](https://github.com/atom/github/pull/1503)
+* PR status badge. -> `Open`.
+* Branches -> `master` < `aw/rfc-pr-list`
+* `Commits` with count, links to .com (for now)
+* `Checks` with count, links to .com (for now)
+* `Files changed` with count, links to .com (for now)
+  * Plus a "Checkout" button to fetch (if necessary) and check out the pull request. Only enabled if the current pull request is not the current one.
+* `Conversation` with comment count.
```

Comment:
When making the mockup I had a deja-vu 😄 and sure enough, there is a closed PR that included the conversation and comments in a panel (now dock item): https://github.com/atom/github/pull/494

It was ok for small PRs. But for PRs that had a long description and lots of long comments, it just felt too cramped and you scroll forever. Here a gif that tries to simulate it with this PR: https://github.com/atom/github/pull/1370

![timeline](https://user-images.githubusercontent.com/378023/41035025-60f1fbd8-69c6-11e8-8359-f47da4ae9e44.gif)

We then concluded that it's better to open a center pane for the conversation. So not sure if we should try again having the conversation in a dock item? An alternative might be to **only** have the description, but not the comments. If you want the read the whole timeline with comments, click on `Conversation` and it links to .com.

Reply:
Hahaha! Oops.

> We then concluded that it's better to open a center pane for the conversation.

👍